federator/src/lib/TransactionSender.js

```diff
@@ -36,19 +36,11 @@ module.exports = class TransactionSender {
         return this.getEthGasPrice();
     }
 
-    async getGasLimit(rawTx) {
-        const chainId = await this.getChainId();
-        let estimatedGas = await this.client.eth.estimateGas({
-            gasPrice: rawTx.gasPrice,
-            value: rawTx.value,
-            to: rawTx.to,
-            data: rawTx.data,
-            from: rawTx.from
-        });
-
-        if (chainId >= 30 && chainId <= 33) {
-            estimatedGas = 200000;
-        }
+    async getGasLimit() {
+        // Gas estimation does not work correctly on RSK and after the London harfork, neither is working on Ethereum
+        // example https://etherscan.io/tx/0xd30d6cf428606e2ef3667427b9b6baecb2f4c9cbb44a0c82c735a238ec8f72fb
+        // To fix it, we decided to use a hardcoded gas estimation
+        const estimatedGas = 250000;
```

Comment:
I'd maybe move this to the top of the file as a constant and just return it here (to not recreate it on each call)